apis/js-apis/common-examples/record-attributions.md

```diff
@@ -31,8 +49,13 @@ nostojs(function(api) {
   .done()
 });
 ```
+In the above example,
+- `vp` specifies the type of event and it corresponds to View Product
+- `12345678` specifies the target and it corresponds to the ID of the product that's being viewed
+- `frontpage-nosto-1` specifies tge reference and it corresponds to the placement that hosted the product that's being clicked
 
-(2) Attributing a placement click to a `cc` event
+
+2. Attributing a placement click to a `cc`  (Custom Campaign) event
```

Comment:
Can you explain what a custom campaign is? I'm not sure what it is either 😅

Reply:
Done